network-api/networkapi/wagtailpages/utils.py

```diff
@@ -31,7 +31,9 @@ def get_descendants(node, list, depth=0, max_depth=2):
     discovery pass of all menu-listable children to some root node.
     '''
     if (depth <= max_depth):
-        title = node.specific.header if node.specific.header else node.title
+        title = node.title
+        if hasattr(node.specific, 'heading') and node.specific.heading:
```

Comment:
"if exists, and non-empty, use it"

Reply:
hm, it should be header.  As for the simplification, would `getattr(node.specific, 'header', node.title)` then work?